AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/AnyModelIntegrationTests.swift

```diff
@@ -8,10 +8,8 @@
 import XCTest
 import Foundation
 import AWSAPIPlugin
-import AWSPluginsCore
-
+@testable import APIHostApp
 @testable import Amplify
-@testable import AmplifyTestCommon
```

Comment:
AmplifyTestCommon does not exist as a package or framework for the integration host app to use, the files in AmplifyTestCommon folder has not been touched and they were added to the APIHostApp target so they can be used by the integration tests